includes/wikia/models/DesignSystemGlobalNavigationModel.class.php

```diff
@@ -32,7 +32,12 @@ public function getData() {
 				'header' => [
 					'type' => 'link-image',
 					'href' => $this->getHref( 'fandom-logo' ),
+					// 'image' is deprecated use 'image-data' instead
 					'image' => 'wds-company-logo-fandom-powered-by-wikia',
+					'image-data' => [
+						'type' => 'wds-svg',
+						'name' => 'wds-company-logo-fandom-powered-by-wikia',
+					],
 					'title' => [
 						'type' => 'text',
 						'value' => 'Fandom powered by Wikia'
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "value" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)